crates/host/src/oci.rs

```diff
@@ -281,8 +282,12 @@ impl Fetcher {
     }
 
     /// Used to set additional CA paths that will be used as part of fetching components and providers
-    pub fn with_additional_ca_paths(mut self, additional_ca_paths: &Vec<PathBuf>) -> Self {
-        self.additional_ca_paths.clone_from(additional_ca_paths);
+    pub fn with_additional_ca_paths(mut self, paths: &[impl AsRef<Path>]) -> Self {
+        self.additional_ca_paths = paths
+            .into_iter()
+            .map(AsRef::as_ref)
+            .map(PathBuf::from)
+            .collect();
```

Comment:
I swear I tried this exact same thing, and my compiler was unhappy about it, but I'm glad you were able to nail it! 👍

Reply:
Yeah I wonder if it was `iter()`or `into_iter()`or something -- I could probably get away with `iter()` I wonder...